src/widgets/docsViewer.js

```diff
@@ -18,6 +18,8 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+imports.gi.versions.WebKit2 = '4.0';
```

Comment:
Curious, is this related?

Reply:
Yes, needed for the master runtime, otherwise WebKit will default to GTK 4, ideally we just bundle libhandy 1.6 and there is no need to use the `master` runtime.

Reply:
This is no longer needed.